straxen/analyses/event_display.py

```diff
@@ -36,7 +36,26 @@
                               )
 
 
-@straxen.mini_analysis(requires=('event_info',))
+def _scatter_rec(event,
+                 recs=('mlp', 'cnn', 'gcn'),
+                 scatter_kwargs=None):
+    """Convienent wrapper to show three algorithms for xenonnt"""
+    assert len(recs) <= 3, "I only got three markers/colors"
+    if scatter_kwargs is None:
+        scatter_kwargs = {}
+    scatter_kwargs.setdefault('s', '100')
+    shapes = ('v', '^', '>', '<')
+    colors = ('brown', 'orange', 'lightcoral')

```

Comment:
For color you could also use the default matplotlib color cycle if you like `plt.rcParams['axes.prop_cycle'].by_key()['color']` those are eight colors I think.

Reply:
Yeah, thanks, I don't think those contrast well against the green background

Reply:
Would be cool to have a matplotlib feature which could detect this and change the color automatically. Maybe if you have time ;)

Reply:
But anyhow maybe it does not hurt to add two more colors and shapes already now, just in case.

Reply:
True